pages/_story.vue

```diff
@@ -37,8 +41,18 @@
           </div>
         </div>
       </div>
-      <div v-else>
-        <p> No image found for this chapter. Does the chapter tag for this story have an image key? e.g. :::Chapter{headline="Name of my chapter" image="chapimg.png"} </p>
+      <!-- Video from youtube -->
+      <div v-else-if="chapter.props.video" class="w-2/3 bg-white rounded">
+        <iframe class="object-contain w-full h-full max-w-full max-h-full mx-auto" v-bind:src="'https://www.youtube.com/embed/' + chapter.props.video" title="YouTube video player" frameborder="0"></iframe>
```

Comment:
```suggestion
        <iframe class="object-contain w-full h-full max-w-full max-h-full mx-auto" :src="'https://www.youtube.com/embed/' + chapter.props.video" title="YouTube video player" frameborder="0" />
```

Again, Github Actions was complaining about this (I tested this and it still works; v-bind seems to be optional here)